package/resiot/control/prerm

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+/etc/init.d/helium_gateway stop
```

Comment:
remove this line.. auto upgrades when the system tries to stop during the upgrade